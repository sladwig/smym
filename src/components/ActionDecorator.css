.action-decorator {
    padding: 14px 19px;
    border-radius: 28px;
    background: blue;
}