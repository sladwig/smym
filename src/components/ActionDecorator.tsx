import React, { ReactElement } from 'react';
import './ActionDecorator.css';
import { Icon } from './Icon';
import { ReactComponent as CrossSvg } from '../images/cross-icon.svg';

interface IProps {
    children: ReactElement;
}

export const ActionDecorator = ({ children }: IProps) => {
    return (
        <span className="action-decorator">
            {children}
            {/* <Icon name="cross" /> */}
            <CrossSvg />
        </span>
    );
};
