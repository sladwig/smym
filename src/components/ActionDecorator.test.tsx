import { render } from '@testing-library/react';
import React from 'react';
import { ActionDecorator } from './ActionDecorator';

describe('ActionDecorator', () => {
    test('renders', () => {
        const { container } = render(<ActionDecorator />);
        expect(container).toMatchSnapshot();
    });
});
