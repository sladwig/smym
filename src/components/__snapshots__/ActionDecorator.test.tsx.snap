// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActionDecorator renders 1`] = `<div />`;
